### PR TITLE
Product listing page added

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (556)
Screenshot (557)
Screenshot (558)

### CODE REVIEW

**Review by ajit-sharmaGH — APPROVED** [review]
everything looks good! 
Try to create small PR!

File: CSS-files-e-com/login.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.eye-btn{
+    position: absolute;
+    top: 4.1rem;
+    right: .5rem;
+    border: none;
+}
+.btn-login{
+    padding: .4rem;
+    border-radius: .4rem;
+    /* outline: none; */
```
Comment (raghav6202) [inline CSS-files-e-com/login.css:37]: Can remove commented code if not required

File: Components-E-Com/Home-Page/index.html [hunk 2]
```diff
@@ -51,41 +51,41 @@ <h2 class="text-align-center top-margin-7rem"> Featured Categories </h2>
...
-                <div class="card-container">
-                    <main class="main-body wrap-up">
-                        <h3 class="changed-position card-title bolder">Clothing</h3>
-                        <picture class="background-img">
-                            <img src="/Assets/category-1.jpg" alt="image" class="basic-image">
+                <div class="card-contain-e-land">
+                    <main class="wrap-up main-body-e-land">
+                        <h3 class="bolder change-pos-e-land card-title-e-land">Clothing</h3>
+                        <picture class="background-img-e-com">
+                            <img src="/Assets/category-1.jpg" alt="image" class="basic-image-e-land">
```
Comment (raghav6202) [inline Components-E-Com/Home-Page/index.html:58]: try to stick to two class names if possible

**Review by raghav6202 — APPROVED** [review]
Looks good to merge

File: CSS-files-e-com/login.css (hunk 1 above)
Comment (krate27) [reply CSS-files-e-com/login.css:37]: It is for my personal reference in future, have kept this keeping in mind we will polish projects,

**Pull request merged**